modules/search/class.jetpack-search.php

```diff
@@ -248,6 +250,11 @@ public function load_assets() {
 		}
 	}
 
+	public function load_and_initialize_tracks() {
+		define( 'W_JS_VER', 60 ); // W_JS_VER
+		wp_enqueue_script( 'jetpack-instant-search-tracks', '//stats.wp.com/w.js', array(), W_JS_VER, true ); 
```

Comment:
It would be important to use `jp-tracks` slug here so that this doesn't get enqueued twice.

Reply:
Good call!